internal/worker/rundynamic.go

```diff
@@ -163,6 +205,10 @@ func RunDynamicAnalysis(ctx context.Context, pkg *pkgmanager.Pkg, sbOpts []sandb
 		return DynamicAnalysisResult{}, err
 	}
 
+	if err := addSSHKeysToSandbox(ctx, sb); err != nil {
+		return DynamicAnalysisResult{}, err
```

Comment:
I'm not sure if we should fail the whole of dynamic analysis if adding SSH keys fails. Perhaps we should just log the error and proceed without SSH keys. Thoughts @calebbrown?

Reply:
Ah yeah just logging and proceeding without SSH keys makes sense to me.

Reply:
Done.

Reply:
Thanks!